src/ui/public/saved_objects/saved_object.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
+
+export class SavedObject {
+  constructor(client, attributes) {
+    this.client = client;
```

Comment:
Can we use `this._client` and `this._attributes` so that external code doesn't start accessing `savedObject.attributes` directly?